Xamarin.Forms.Platform.iOS/CollectionView/ListSource.cs

```diff
@@ -38,7 +38,7 @@ public void Dispose()
 					throw new ArgumentOutOfRangeException(nameof(indexPath));
 				}
 
-				return this[indexPath.Row];
+				return this[indexPath.Item];
```

Comment:
Build is failing. The error:
"Xamarin.Forms.Platform.iOS \ CollectionView \ ListSource.cs (41.17): Error CS1503: Argument 1: cannot convert from 'System.nint' to 'Foundation.NSIndexPath'".

Something like this:
`return this[(int)indexPath.Item];`
Will fix the Build problem.